cli/internal/cmd/configfetchmeasurements_test.go

```diff
@@ -249,23 +231,35 @@ func TestConfigFetchMeasurements(t *testing.T) {
 	})
```

Comment:
The actual value of `signature` some lines above isn't used anymore. Maybe replace with a dummy value to make it obvious.